shell/src/main/java/alluxio/cli/fs/command/LsCommand.java

```diff
@@ -89,6 +89,14 @@ public final class LsCommand extends AbstractFileSystemCommand {
     TIMESTAMP_FIELDS.put("lastModificationTime", URIStatus::getLastModificationTimeMs);
   }
 
+  private static final Option EXCLUDE_MOUNT_INFO =
+      Option.builder()
```

Comment:
I have a high level question on how you intend to use this improvement. Are you going to specify USER_HDFS_CLIENT_EXCLUDE_MOUNT_INFO_ON_LIST_STATUS=true in your environment? I mean, this can totally work, but maybe in the future we let users have the improvements by default.

Yes, set `USER_HDFS_CLIENT_EXCLUDE_MOUNT_INFO_ON_LIST_STATUS=true`. In fact, I wanted to be more radical. I want to it is default true, and in our environment, it is default true.
.But if it defaults  false, it will be safer.

https://github.com/Alluxio/alluxio/pull/15578 and https://github.com/Alluxio/alluxio/pull/16388 are two ongoing attempts to make MountTable.resolve() faster, so you might not need to disable that call at all. You might wanna stay tuned or leave your comments on those PRs :)
Ok, I will stay tuned or leave your comments on those PRs;